templates/tab/js/default/src/components/Tab.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Welcome } from "./sample/Welcome";
```

Comment:
Is there any code change in plugin to set this env variable?

Reply:
No. We have already set the variable.